msi/docs/build-msi-locally.md

```diff
@@ -0,0 +1,139 @@
+Should you ever need to rebuild a Jenkins MSI on your Windows machine, here is a way to do it.
+
+# Pre-requisites
+
+## Jenkins WAR file
+
+First of all, you should get the Jenkins war file that will be inside that MSI file.
+You can get it from the official Jenkins website or from the Jenkins update center.
+I will use the official Jenkins website in this article.
```

Comment:
It's a doc not an article or a blog post, not sure I'd use "I" and this formulation. Idem for the following "for me".
(Subjective)

Reply:
You're right.